wiki/People/Completionists/en.md

```diff
@@ -0,0 +1,30 @@
+# Completionists
+
+Completionists are players who take on the extremely daunting task of playing **every [Ranked](/wiki/Beatmap/Category#ranked) [beatmap](/wiki/Beatmap)** of a given game mode. With the amount of Ranked beatmaps increasing every day, this feat is becoming progressively more difficult to achieve, requiring insane amounts of dedication and perseverance.
+
+As of August 2023, this achievement is recognised through a special [profile badge](/wiki/Community/Profile_badge)
+
+![osu!](img/osu.png "osu! completionist badge") ![osu!taiko](img/taiko.png "osu!taiko completionist badge") ![osu!catch](img/catch.png "osu!catch completionist badge") ![osu!mania](img/mania.png "osu!mania completionist badge")
+
+## Completion criteria
+
+In order to be considered a completionist, a player must play every existing Ranked beatmap of a given game mode with no [difficulty reduction mods](/wiki/Gameplay/Game_modifier#difficulty-reduction), this does not include converted beatmaps for [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania). For osu!mania, only 4K and 7K beatmaps are considered for completion.
+
+## Confirmed completionists
+
+| Player | Game mode | Date of completion | Total played beatmaps |
```

Comment:
just want to confirm that you mean "beatmap" and not "difficulty"

Reply:
difficulty, i'll clarify that